platforms/hyperledger-besu/configuration/roles/create/k8_component/templates/azure-storageclass

```diff
@@ -0,0 +1,11 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: {{ sc_name }}
+provisioner: disk.csi.azure.com
+parameters:
```

Comment:
add the parameters in platforms/shared/charts/storage_class/values.yaml as shown in the screen shot. this template file is not required

Reply:
@suvajit-sarkar thank you for following up, I had actually tried these steps already before trying the work around, but the creation of the storage class failed.

I just tried again and the Azure storage classes are still failing to create when I use the new charts.

Reply:
The issue is with allowedTopologies, make it empty in values.yaml for azure, it should work.